gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.ts

```diff
@@ -70,6 +74,21 @@ export class ApplicationGeneralComponent implements OnInit {
               this.initialApplication.settings?.app?.client_id ? this.initialApplication.settings.app.client_id : undefined,
             ),
           }),
+          OpenIDForm: new FormGroup({
+            client_id: new FormControl(
+              this.initialApplication.settings?.oauth?.client_id ? this.initialApplication.settings.oauth.client_id : undefined,
+            ),
+            client_secret: new FormControl(
+              this.initialApplication.settings?.oauth?.client_secret ? this.initialApplication.settings.oauth.client_secret : undefined,
+            ),
+            grant_types: new FormControl(
+              this.initialApplication.settings?.oauth?.grant_types ? this.initialApplication.settings.oauth.grant_types : undefined,
+              [Validators.required],
```

Comment:
What do you think about not init form OpenIDForm and OAuth2Form if we don't have the right initialApplication.type? 

because the `required` here doesn't also apply type === SIMPLE ?

Reply:
Done